diffkemp/diffkemp.py

```diff
@@ -113,6 +116,16 @@ def build_c_project(args):
     # Build the project using generated wrapper
     check_call(make_target_args, env=environment)
 
+    # Run LLVM IR simplification passes if the user did not specified
+    # to not override project optimization.
+    if not args.no_opt_override:
+        # Run llvm passes on created LLVM IR files.
+        with open(db_filename, "r") as db_file:
+            for line in db_file:
+                if line.startswith("o:"):
```

Comment:
Nit: this saves one level of indentation:
```suggestion
            for line in [l for l in db_file if l.startswith("o:")]:
```
You could even meld the next line inside but that would be probably less readable due to line breaks:
```suggestion
            for line in [l.split(":")[1].rstrip() for l in db_file if l.startswith("o:")]:
```

Reply:
I used the first suggestion, the second was too long for one line.